src/main/java/io/confluent/connect/elasticsearch/Mapping.java

```diff
@@ -66,27 +58,21 @@ public static void createMapping(JestClient client, String index, String type, S
   }
 
   /**
-   * Check the whether a mapping exists or not for a type.
-   * @param client The client to connect to Elasticsearch.
-   * @param index The index to write to Elasticsearch.
-   * @param type The type to check.
-   * @return Whether the type exists or not.
+   * Get the JSON mapping for given index and type. Returns {@code null} if it does not exist.
```

Comment:
The test code was retrieving the mapping correctly, while the existence check here was wrong. So I thought it best to just have a retrieval method here such that `null` return indicates missing.